packages/push/src/instruction/bool.rs

```diff
@@ -22,9 +22,6 @@ pub enum BoolInstruction {
     // BoolFromFloat,
 }
 
-#[derive(thiserror::Error, Debug, Eq, PartialEq)]
-pub enum BoolInstructionError {}
```

Comment:
Why was this removed?

Reply:
Because `BoolInstructionError` would be for specific errors caused by boolean instructions. But all boolean operators (at least that we have at the moment) can never fail.

This is different from `IntInstructionError` since integer instructions can generate errors like overflow.

it's dimly possible that we'll want to bring it back/keep it, but I couldn't think of anything special for booleans that might fail.

Reply:
Ok LGTM